### PR TITLE
code cleanup src/

### PR DESCRIPTION
**Pull request by nikkimoteva**

### CODE REVIEW

File: src/Authentication/Auth.js [hunk 1]
```diff
@@ -81,10 +66,10 @@ function useAuthProvider() {
       .then(() => {
         console.log("Stored info in backend");
         const profile = credentials.getBasicProfile();
-        const id = profile.getId(); // Do not send to your backend! Use an ID token instead.
+        const id = profile.getId();
```
Comment (KianShah) [inline src/Authentication/Auth.js:69]: keep these comments. They were included in google starter code

File: src/Authentication/LoginModal.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default class LoginModal extends React.Component {
             clientId="296036318202-uraiim5u0cf5qpqhujl3aaj1kniuu41e.apps.googleusercontent.com"
             buttonText="Login with Google"
             onSuccess={this.props.signin}
-            onFailure={(res) => console.log(res)}
+            onFailure={(err) => console.log(err)}
```
Comment (KianShah) [inline src/Authentication/LoginModal.js]: console.err

File: src/BaseRouter.js [hunk 3]
```diff
@@ -9,9 +9,6 @@ import ContactUs from "./ContactUs/Contact";
 import React, {lazy, Suspense} from "react";
 
 export default function BaseRouter() {
-
-  // lazy load it because we don't want ppl to have to wait for this to load just to view the page
```
Comment (KianShah) [inline src/BaseRouter.js:13]: Keep

**Review by robertmzm — APPROVED** [review]
Reviewed

**Pull request merged**